### DISCUSSION TITLE
Elecrow Pre-build Deskhop

### DISCUSSION THREAD

**Discussion by lylejantzi3rd** [hrvach/deskhop#167] [General]
Hello! I bought a pre-built Deskhop from Elecrow and I can't get it to work. 

The mouse stays on computer B and never switches to computer A. The keyboard stays on computer B and never switches to computer A. I can hit ctrl+caps lock to disable the keyboard. Entering config mode (Left Ctrl + Right Shift + C + O) doesn't work. 

Is there some initial setup I'm missing?

**Comment by hrvach** [comment]
First try upgrading to the latest firmware, but I suspect there might be some issue with the box. Do both leds flash when you plug in the keyboard and mouse, like this:

![blink-blink](https://github.com/user-attachments/assets/585a2819-99a6-4280-9833-040d33d2c839)

?

Check https://github.com/hrvach/deskhop/wiki/Troubleshooting for more debugging tips.

**Reply by lylejantzi3rd** [reply]
Nope. Neither of the lights flash. One comes on solid. I'll upgrade the firmware and let you know how it goes. Thanks!

**Comment by hrvach** [comment]
Download the latest .uf2, first do one side then the other (press the button while plugging it in, then just copy the file to the usb drive that appears). Let me know if anything changes.

**Reply by lylejantzi3rd** [reply]
It worked! The lights flash as expected and I can control both computers. I can even open the config. It looks like elecrow didn't flash either of the raspberry PIs.

Thanks for the help! This is going to make my life so much better, you have no idea.

**Reply by Al-Infinitum** [reply]
Thank you, I experienced the exact same issue and flashing the latest firmware to both sides immediately fixed the problem.

Quick note for others: "the button" to press is on the board itself, users need to open the case and press the button near the micro-USB port while connecting to the machine with the downloaded uf2 file. This is all described in the firmware update section, but I skimmed too lightly and missed the alternate option. When I was unable to enter config mode with the keyboard shortcut, I thought a firmware update would be impossible.

Thanks for the help Hrvoje, and to lylejantzi3rd for raising the issue!
